lms/static/scripts/frontend_apps/components/dashboard/GradeIndicator.tsx

```diff
@@ -72,23 +79,25 @@ export default function GradeIndicator({
     : 0;
 
   return (
-    <div
-      className="inline-block relative"
-      onMouseOver={showPopover}
-      onMouseOut={hidePopover}
-      // Make element focusable so that the popover can be shown even when
-      // interacting with the keyboard
-      onFocus={showPopover}
-      onBlur={hidePopover}
-      role="button"
-      tabIndex={0}
-      data-testid="container"
-    >
-      <GradeStatusChip grade={grade} />
+    <div className="relative">
+      <button
```

Comment:
Even though a button is used, there's no visual difference, because tailwind removes all default styles.